engine/src/main/java/org/terasology/rendering/opengl/PBO.java

```diff
@@ -30,7 +30,8 @@
  * Created by manu on 15.03.2015.
  */
 public class PBO {
-    private int pboId;
+    private ByteBuffer resultBuffer = null;
```

Comment:
Please, can you let me know your reasoning regarding this and the associated changes? What does this additional buffer do?